client/components/code_editor/code-editor-controller.js

```diff
@@ -123,6 +123,17 @@ explorer.components.code_editor.CodeEditorCtrl = function(
   };
 
   /**
+   * Options for the CodeMirror editor.
+   * @export {!Object}
+   */
+  this.editorOptionsSQLReadOnly = {
+    lineWrapping: true,
+    lineNumbers: true,
+    mode: 'text/x-sql',
+    readOnly: true
```

Comment:
The cursor is intentional; I want to make it easy to focus the control and CTRL-A to select everything, and this gesture doesn't work without a cursor.

Added a tooltip to the button.

I agree it would be awesome, but out of scope for this fix.